documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,63 @@
+# Get Bootchart from a Clear Container#
+
+The following steps describe how to get a bootchart from a Clear Cotainers.
+Currently systemd is the default init process in the Clear Containers mini-os
+system. Systemd provides a tool called systemd-bootchart that  that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart##
+
+To get a bootchart from a Clear Container you must vefiry the following 
+steps before:
+
+### Prepare your Clear Container image ###
+
+The file data/cc-bootchart.conf can be used get a bootchart graph.
+
+The file must be installed Clear-Container image using systemd and systemd 
+bootchart.
+
+The file data/cc-bootchart can be installed with the configure option
+--with-cc-image-bootchart-config-dir=DIR  where DIR is a directory
+to any of the following paths:
+
+```
+/etc/systemd/bootchart.conf
+/etc/systemd/bootchart.conf.d/*.conf
+/run/systemd/bootchart.conf.d/*.conf
+/usr/lib/systemd/bootchart.conf.d/*.conf
+```
+
+Note : Any any of the paths above must inside the container image.
+
+### Configure Clear Containers to boot with bootchart ###
+
+Make sure that you kernel use as init proccess as systemd-bootchart
+
+To speficify the init binary modify the file vm.json the file:
+
+Append to the "parameters" json node "init=PATH-TO-SYSTEMD-BOOTCHART"
```

Comment:
This is an interesting one: if the user is using a packaged version of CC then yes, they need to update `vm.json`, but if they have a git clone, it would be better (and simpler) to modify `data/kernel-cmdline` and then run the build.

But also, we are currently explicit about the init daemon and `data/kernel-cmdline` does currently specify `init=`, so although I think the kernel will honour the last specified value for a variable it would probably be safer (and less confusing) to say "change `init=/usr/lib/systemd/systemd` to `init=...`".